RIBs.podspec

```diff
@@ -11,5 +11,5 @@ RIBs is the cross-platform architecture behind many mobile apps at Uber. This ar
   s.source           = { :git => 'https://github.com/uber/RIBs.git', :tag => 'v' + s.version.to_s }
   s.ios.deployment_target = '8.0'
   s.source_files = 'ios/RIBs/Classes/**/*'
-  s.dependency 'RxSwift', '~> 4.5.0'
+  s.dependency 'RxSwift', '~> 5.0'
```

Comment:
It looks like RIBs needs to explicitly depend on `RxRelay` as well.

Reply:
Good catch. Is there a specialized testing facility for Cocoapods that we could use to verify that specs are complete or is it just "try to xcodebuild"?

Reply:
Yeah, you can run `pod lib lint`.
https://stackoverflow.com/a/32328089